src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,23 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:  # type: ignore
+        """
+        Populate package manifest with metadata as per EIP
```

Comment:
This does not populate the manifest, so I would change this doc.
This just reads the value from the config manager.

Try something like

```
Metadata about the active project as per EIP ...
Use when publishing your package manifest.
```